Src/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -49,6 +49,16 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string LoginHint { get; set; }
 
         /// <summary>
+        /// Gets or sets the Include Granted Scopes to determine if this authorization request should use

```

Comment:
Fixed, I probably had just copied in the variable name and added spaces because I'm a special kind of lazy sometimes when it gets late...